Yafc/Windows/WelcomeScreen.cs

```diff
@@ -202,24 +208,17 @@ protected override void BuildContents(ImGui gui) {
     }
 
     private void ProjectErrorMoreInfo(ImGui gui) {
-        gui.allocator = RectAllocator.LeftAlign;
```

Comment:
I figured out why the allocator needs to be changed from Stretch to LeftAlign. Hovering the link stretches the underline across the entire tooltip:
![image](https://github.com/user-attachments/assets/6e180360-c36f-4576-92b7-8e146b0f21bf)

Reply:
Alright, so I need to put the allocator back, from what I understand.

Reply:
Addressed in https://github.com/shpaass/yafc-ce/pull/351/commits/e34eb9445e30432085aa2bfa91b60827a84d6746